src/manager/node.rs

```diff
@@ -305,7 +314,16 @@ where
 
                             if OP::OperatorState::has_tables() {
                                 if let Some(snapshot) = &self.latest_snapshot {
+                                    #[cfg(feature = "metrics")]
```

Comment:
This part of the code will most likely be removed in the coming future. So best to remove everything regarding the last_checkpoint_restore_timestamp metric